packages/server/src/interfaces/agent-service.interface.ts

```
@@ -26,7 +26,7 @@ export interface IAgentService {
   handleUserRequest(
     agent: AgentSystem | IAgent,
     userRequest: MessageRequest
-  ): Promise<AgentExecutionResponse>;
+  ): Promise<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)